### PR TITLE
Update Society Public Safety

### PR DESCRIPTION
**Pull request by eneemann**
Updates for SGID changes to PSAPLocations, PSAPBoundaries

More updates to follow:

- [x] Add new PSAP update date
- [x] Add Law Enforcement Boundaries "package" to public safety page
- [x] Add image for new "package"
- [x] Update outdated Abstract on public safety page
- [x] Update outdated Description on [society overview](https://gis.utah.gov/data/society/)

Once this is merged, the bot pr for the sgid index can be merged.

### CODE REVIEW

**Comment by eneemann** [comment]
I think the preview looks good, so I'm opening this PR up for review.

File: data/society/public-safety/index.html [hunk 1]
```diff
@@ -109,6 +118,20 @@ <h3 id="LawEnforcement">Law Enforcement</h3>
     info=page.LawEnforcement
     %}
 </div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="LawEnforcementBoundaries">Law Enforcement Boundaries</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/Lawboundaries.png %}" alt="Law Boundary icon" loading="lazy" />
+    <p>Society.LawEnforcementBoundaries contains jurisdictional boundaries for law enforcement agencies in Utah. Law enforcement boundaries indicate which law enforcement agency has the primary responsibility in a specific area. These boundaries are also used in the Next-Generation 911 system for recommending which agency should respond to a specific location.</p>
```
Comment (steveoh) [inline data/society/public-safety/index.html:127]: Nitpick: These are the internal table names. We've been moving away from those by using the normal name.

**Review by steveoh — APPROVED** [review]

File: data/society/public-safety/index.html (hunk 1 above)
Comment (eneemann) [reply data/society/public-safety/index.html:127]: @steveoh Okay, I guess I was just trying to align it with the way the rest of the page's tables were named.  Should I change all of them to just 'Fire Stations', 'PSAP Boundaries', 'Law Enforcement Boundaries', etc.?

Comment (steveoh) [reply data/society/public-safety/index.html:127]: Yah that would be an improvement

**Pull request merged**